### ISSUE TITLE
Node Bug / Problem

### ISSUE THREAD

**Issue by Hannn3x** [Shardeum/shardeum-bug-reporting#926]
I am having trouble running node, and my node is not running properly.

My node status says "No SHM Staked" even though I previously staked 75-90 SHM.

All this started happening when I reinstalled my Node because previously the node status was always "Stand by & Stopped"
![6805abc0-3067-453a-8161-aa20957d81fc](https://github.com/user-attachments/assets/04b0f476-ef03-4bf0-8c32-7773880590ec)
![730f359c-e422-49a6-bbbc-b76c014f3f49](https://github.com/user-attachments/assets/12bd9e27-a8b6-47ac-9035-00d32ac5a31d)

**Comment by HT-Moh** [comment]
Yes I have the same issue if you go to force remove stake you see it 
![image](https://github.com/user-attachments/assets/85a0a03d-2bd4-4865-a350-763d03158f01)
